joomla/joomla-cms/CVE-2018-6376.yaml

```diff
@@ -0,0 +1,8 @@
+title: SQLi vulnerability in Hathor postinstall message
+link: https://developer.joomla.org/security-centre/722-20180105-core-sqli-vulnerability.html
+cve: CVE-2018-6376
+branches:
+    master:
+        time:     2018-1-30
+        versions: ['>=3.7.0', '<3.8.3']
```

Comment:
this version range is wrong. The link you gave says that 3.8.3 is also vulnerable, and the fix is part of 3.8.4.

I haven't checked all other ones